apps/passport-client/components/shared/PCDCard.tsx

```diff
@@ -124,6 +128,18 @@ function CardFooterImpl({
   );
 }
 
+function TicketCardBody({ pcd }: { pcd: EdDSATicketPCD }) {
+  const ticketType = useMatchedTicketType(pcd);
+
+  if (ticketType && ticketType.ticketGroup === KnownTicketGroup.Devconnect23) {
+    return <DevconnectCardBody pcd={pcd} />;
+  }
+
+  const Component = EdDSATicketPCDPackage.renderCardBody;
+
+  return <Component pcd={pcd} />;
+}
+
```

Comment:
I half-agree. This exists in order to show the special QR code for Devconnect check-in, and so it can be removed after Devconnect.

I'm working on an assumption that `EdDSATicketPCD` is providing generic ticket functionality, and `passport-client` is doing a special-case override because it knows that there's this Devconnect thing, for which it also has some special routes. Having `EdDSATicketPCD` generate a QR code that points to the special Devconnect routes in `passport-client` would mean that they both have to know about Devconnect, rather than just `passport-client`.